styles/globals.css

```diff
@@ -1,3 +1,125 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+html {
+  scroll-behavior: smooth;
+}
+
+.parallax {
```

Comment:
let's use css modules for this, and for the positioning and styling i'd use tailwind to keep things consistent